tendermint/src/node/info.rs

```diff
@@ -33,7 +33,7 @@ pub struct Info {
 }
 
 /// Protocol version information
-#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize, Default)]
```

Comment:
Are zeroes for all of these version values semantically meaningful?

Reply:
Removed.